airflow/providers/google/cloud/transfers/vertica_to_gcs.py

```diff
@@ -0,0 +1,106 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import decimal
+
+import datetime
+import sys
+
+import time
+
+from airflow.providers.vertica.hooks.vertica import VerticaHook
+from airflow.providers.google.cloud.transfers.sql_to_gcs import BaseSQLToGCSOperator
+from airflow.utils.decorators import apply_defaults
+
+PY3 = sys.version_info[0] == 3
+
+
+class VerticaToGoogleCloudStorageOperator(BaseSQLToGCSOperator):
```

Comment:
```suggestion
class VerticaToGCSOperator(BaseSQLToGCSOperator):
```
We try to use the GCS abbreviation because it is widely recognized.

Reply:
+1